common/containers/Tabs/GenerateWallet/components/EnterPassword.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Field, reduxForm } from 'redux-form';
+import translate from 'translations';
+import { genNewKeystore } from 'libs/keystore';
+import PasswordInput from './PasswordInput';
+
+// VALIDATORS
+const minLength = min => value => {
+  return value && value.length < min
+    ? `Must be ${min} characters or more`
+    : undefined;
+};
+const minLength9 = minLength(9);
+const required = value => (value ? undefined : 'Required');
+
+class EnterPassword extends Component {
+  static propTypes = {
+    // state
+    generateWalletPassword: PropTypes.object,
+    showPassword: PropTypes.bool,
```

Comment:
What I think the original author intended to indicate (This file is just another component that was broken into 2) is that these are application state from `mapStoreToProps` rather than action functions from `mapDispatchToProps`. I just left the comment for posterity.